docs/basics/101-139-hostingservices.rst

```diff
@@ -160,7 +162,163 @@ Should you employ GitHub workflows, for example, a token without "workflow" scop
 
     [remote rejected] (refusing to allow a Personal Access Token to create or update workflow `.github/workflows/benchmarks.yml` without `workflow` scope)]
 
+.. _gitlab:
 
+Creating a sibling on GitLab
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:term:`GitLab` is an open source Git repository hosting platform, and many institutions and companies deploy their own instance.
+This short walk-through demonstrates the necessary steps to create a GitLab sibling, and the different options GitLab allows for when creating siblings recursively for a dataset hierarchy.
+
+Step 1: Configure your site
+"""""""""""""""""""""""""""
+
+As a first step, users will need to create a configuration file following the format of `python-gitlab <https://python-gitlab.readthedocs.io/en/stable/cli-usage.html#configuration-file-format>`_.
+This configuration file is typically called ``.python-gitlab.cfg`` and placed into a users home directory.
+It contains one section per GitLab instance, and a ``[global]`` section that defines the default instance to use.
+Here is an example:
+
+.. code-block:: bash
+
+   $ cat ~/.python-gitlab.cfg
+    [global]
+    default = my-university-gitlab
+    ssl_verify = true
+    timeout = 5
+
+    [my-university-gitlab]
+    url = https://gitlab.my-university.com
+    private_token = <here-is-your-token>
+    api_version = 4
+
+    [gitlab-general]
+    url = https://gitlab.com
+    api_version = 4
+    private_token = <here-is-your-token>
+
+Once this configuration is in place, ``create-sibling-gitlab``'s ``--site`` parameter can be supplied with the name of the instance you want to use (e.g., ``datalad create-sibling-gitlab --site gitlab-general``).
+Ensure that the token for each instance has appropriate permissions to create new groups and projects under your user account using the GitLab API.
+
+.. figure:: ../artwork/src/gitlab-token.png
+
+Step 2: Create or select a group
+""""""""""""""""""""""""""""""""
+
+GitLab's organization consists of *projects* and *groups*.
+Projects are single repositories, and groups can be used to manage one or more projects at the same time.
+In order to use ``create-sibling-gitlab``, a user **must** `create a group <https://docs.gitlab.com/ee/user/group/#create-a-group>`_ via the web interface, or specify a pre-existing group, because `GitLab does not allow  root-level groups to be create via their API <https://docs.gitlab.com/ee/api/groups.html#new-group>`_.
```

Comment:
```suggestion
In order to use ``create-sibling-gitlab``, a user **must** `create a group <https://docs.gitlab.com/ee/user/group/#create-a-group>`_ via the web interface, or specify a pre-existing group, because `GitLab does not allow root-level groups to be create via their API <https://docs.gitlab.com/ee/api/groups.html#new-group>`_.
```

Reply:
removed an extra space